apps/react-storybook/src/components/Link.stories.tsx

```
@@ -0,0 +1,23 @@
+import type { Meta, StoryObj } from "@storybook/react-vite";
+
+import { Link } from "@sandwich-ui/react";
```

ESLint

[error] Unable to resolve path to module '`@sandwich-ui/react`'.

(import-x/no-unresolved)